xml/hwbp_registers.xml

```diff
@@ -770,9 +797,7 @@
             Any bits beyond the size of the data access will contain 0.
             </value>
         </field>
-        <field name="0" bits="20" access="WARL" reset="0">
-            This used to be used for timing, but is currently read only 0.
-        </field>
+        <field name="0" bits="20" access="R" reset="0" />
         <field name="size" bits="19:16" access="WARL" reset="0">
```

Comment:
Should this be 18:16?

Reply:
I don't think it matters. Either way we can use that bit for more size encodings, or for a new feature. So that means it's better to be explicit and reduce the size like you suggest.